packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -663,6 +663,13 @@ export interface CloudMapOptions {
    */
   readonly name?: string,
 
+  /**
+   * The service discovery namespace created in this cluster
```

Comment:
It's "the namespace to use for this service".

And the default is "the namespace associated with the cluster"

Reply:
Thought about that, but if a default namespace is not associated to the cluster it will be undefined (does that even matter when specifying the default here?).

Reply:
I would say that if the default here is "the namespace associated with the cluster" and the cluster doesn't have a namespace, then it would be fairly self-evident that you wouldn't get a namespace at all.

HOWEVER if you feel that it's too ambiguous, you can always write "The namespace associated with the cluster, if any" or "..., no namespace if the cluster doesn't have one" or something to that effect.

Reply:
I feel like the second option is out of the norm for the rest of the cdk properties that are used, so I'll stick to what I've got.